hyperspy/gui/egerton_quantification.py

```diff
@@ -391,7 +395,7 @@ def create_interpolation_line(self):
 
     def get_interpolation_range(self):
         axis = self.signal.axes_manager.signal_axes[0]
-        if self.ss_left_value == self.ss_right_value:
+        if np.isnan(self.ss_left_value) or np.isnan(self.ss_right_value):
```

Comment:
I've just tested a zero-width span selector and it doesn't upset HyperSpy. Hence, I suggest to leave it as it is because, otherwise, the interpolation line will expand to the default size when the span selector width becomes 0 which is counterintuitive.